example_configs/directory_backend_config_auth_from_file.json

```diff
@@ -0,0 +1,52 @@
+{
+  "backend": {
+    "module": "medallion.backends.directory_backend",
+    "module_class": "DirectoryBackend",
+    "path": "./example_configs/directory/",
+    "discovery": {
+      "title": "Some TAXII Server",
+      "description": "This TAXII Server contains a listing of...",
+      "contact": "string containing contact information",
+      "host": "http://localhost:5000/"
+    },
+    "api-root": {
+      "title": "",
+      "description": "",
+      "versions": [
+        "taxii-2.0"
+      ],
+      "max-content-length": 9765625
+    },
+    "collection": {
+      "id": "",
+      "title": "",
+      "description": "",
+      "can_read": true,
+      "can_write": true,
+      "media_types": [
+        "application/vnd.oasis.stix+json; version=2.0"
+      ]
+    }
+  },
+  "taxii": {
+    "max_page_size": 100
+  },
+  "auth": {
+    "module": "medallion.backends.auth_memory_backend",
+    "module_class": "AuthMemoryBackend",
+    "users": {
+      "admin": "pbkdf2:sha256:150000$vhWiAWXq$a16882c2eaf4dbb5c55566c93ec256c189ebce855b0081f4903f09a23e8b2344",
+      "user1": "pbkdf2:sha256:150000$TVpGAgEI$dd391524abb0d9107ff5949ef512c150523c388cfa6490d8556d604f90de329e",
+      "user2": "pbkdf2:sha256:150000$CUo7l9Vz$3ff2da22dcb84c9ba64e2df4d1ee9f7061c1da4f8506618f53457f615178e3f3"
+    },
+    "api_keys": {
+      "123456": "admin",
+      "abcdef": "user1"
+    }
+  },
+  "multi-auth": [
+    "basic",
+    "api_key"
+  ],
+  "flask": {}
```

Comment:
What's the purpose of the `"flask"` key/value? I don't see where its used in the source code or documentation about it. (Also present on other example_configs documents)

Reply:
I realized you are using it to store the `SECRET`. Perhaps it makes sense to update the example to be something like:

``` json
"flask": {"SECRET": "CHANGEME"}
```
where you determine its applicable

Reply:
This has been documented in the README. Additionally, it has been removed from the example config files and has a default like the other configuration items.